### ISSUE TITLE
[Bug]: Python: Unable to start a process in spawn mode

### ISSUE THREAD

**Issue by Young-Lord** [termux/termux-packages#10039]
### Problem description

When using multiprocessing library in python3, using `spawn` as start_method, an `ModuleNotFoundError: No module named '_posixshmem'` raised during `process.start()`.

### What steps will reproduce the bug?

```python
import multiprocessing as mp
mp.set_start_method('spawn')
p = mp.Process(target=lambda :print(1))
p.start()
p.join()
```
Run this in python REPL, get the error:
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-4d20206b5cbd> in <module>
      2 mp.set_start_method('spawn')
      3 p = mp.Process(target=lambda :1)
----> 4 p.start()
      5 p.join()

/data/data/com.termux/files/usr/lib/python3.9/multiprocessing/process.py in start(self)
    119                'daemonic processes are not allowed to have children'
    120         _cleanup()
--> 121         self._popen = self._Popen(self)
    122         self._sentinel = self._popen.sentinel
    123         # Avoid a refcycle if the target function holds an indirect

/data/data/com.termux/files/usr/lib/python3.9/multiprocessing/context.py in _Popen(process_obj)
    222     @staticmethod
    223     def _Popen(process_obj):
--> 224         return _default_context.get_context().Process._Popen(process_obj)
    225
    226 class DefaultContext(BaseContext):

/data/data/com.termux/files/usr/lib/python3.9/multiprocessing/context.py in _Popen(process_obj)
    282         def _Popen(process_obj):
    283             from .popen_spawn_posix import Popen
--> 284             return Popen(process_obj)
    285
    286     class ForkServerProcess(process.BaseProcess):

/data/data/com.termux/files/usr/lib/python3.9/multiprocessing/popen_spawn_posix.py in __init__(self, process_obj)
     30     def __init__(self, process_obj):
     31         self._fds = []
---> 32         super().__init__(process_obj)
     33
     34     def duplicate_for_child(self, fd):

/data/data/com.termux/files/usr/lib/python3.9/multiprocessing/popen_fork.py in __init__(self, process_obj)
     17         self.returncode = None
     18         self.finalizer = None
---> 19         self._launch(process_obj)
     20
     21     def duplicate_for_child(self, fd):

/data/data/com.termux/files/usr/lib/python3.9/multiprocessing/popen_spawn_posix.py in _launch(self, process_obj)
     37
     38     def _launch(self, process_obj):
---> 39         from . import resource_tracker
     40         tracker_fd = resource_tracker.getfd()
     41         self._fds.append(tracker_fd)

/data/data/com.termux/files/usr/lib/python3.9/multiprocessing/resource_tracker.py in <module>
     36 if os.name == 'posix':
     37     import _multiprocessing
---> 38     import _posixshmem
     39
     40     _CLEANUP_FUNCS.update({

ModuleNotFoundError: No module named '_posixshmem'
```

### What is the expected behavior?

print(1) without exception

### System information

termux-info:

```
Application version:
0.114
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://mirrors.tuna.tsinghua.edu.cn/termux/termux-packages-24/ stable main
# game-repo (sources.list.d/game.list)
deb https://mirrors.tuna.tsinghua.edu.cn/termux/game-packages-24/ games stable
# science-repo (sources.list.d/science.list)
deb https://mirrors.tuna.tsinghua.edu.cn/termux/science-packages-24/ science stable
# root-repo (sources.list.d/root.list)
deb https://mirrors.tuna.tsinghua.edu.cn/termux/termux-root-packages-24/ root stable
# x11-repo (sources.list.d/x11.list)
deb https://mirrors.tuna.tsinghua.edu.cn/termux/x11-packages x11 main
Updatable packages:
adwaita-icon-theme/x11 41.0 all [upgradable from: 40.1-1]
aircrack-ng/root 2:2021.12.19-f94a3fe3-2 aarch64 [upgradable from: 1:1.6-3]
android-tools/stable 31.0.3p1 aarch64 [upgradable from: 31.0.0p1-2]
apt/stable 2.4.1-2 aarch64 [upgradable from: 2.3.6-1]
aterm/x11 1.0.1-33 aarch64 [upgradable from: 1.0.1-32]
atk/x11 2.38.0 aarch64 [upgradable from: 2.36.0-10]
bash/stable 5.1.16-1 aarch64 [upgradable from: 5.1.8]
binutils/stable 2.38 aarch64 [upgradable from: 2.36.1]
bison/stable 3.8.2-2 aarch64 [upgradable from: 3.7.6]
brotli/stable 1.0.9-1 aarch64 [upgradable from: 1.0.9]
clang/stable 14.0.0 aarch64 [upgradable from: 12.0.1]
cmake/stable 3.23.0-1 aarch64 [upgradable from: 3.21.1]
command-not-found/stable 1.75 aarch64 [upgradable from: 1.62]
coreutils/stable 9.0 aarch64 [upgradable from: 8.32-7]
curl/stable 7.82.0-1 aarch64 [upgradable from: 7.78.0]
dash/stable 0.5.11.5-1 aarch64 [upgradable from: 0.5.11.4]
dbus/stable 1.14.0 aarch64 [upgradable from: 1.12.20-3]
debianutils/stable 5.7 aarch64 [upgradable from: 4.11.2]
desktop-file-utils/x11 0.26-7 aarch64 [upgradable from: 0.26-5]
dialog/stable 1.3-20220117-0 aarch64 [upgradable from: 1.3-20210621-0]
diffutils/stable 3.8 aarch64 [upgradable from: 3.7-3]
dnsutils/stable 9.16.27 aarch64 [upgradable from: 9.16.11]
dpkg/stable 1.21.1-1 aarch64 [upgradable from: 1.20.9-5]
ed/stable 1.18 aarch64 [upgradable from: 1.17]
ethtool/root 5.16 aarch64 [upgradable from: 5.12-1]
exo/x11 4.17.1-3 aarch64 [upgradable from: 4.16.2-1]
ffmpeg/stable 5.0.1 aarch64 [upgradable from: 5.0]
file/stable 5.41-1 aarch64 [upgradable from: 5.41]
findutils/stable 4.9.0-1 aarch64 [upgradable from: 4.8.0]
fontconfig-utils/stable 2.14.0 aarch64 [upgradable from: 2.13.1-6]
fontconfig/stable 2.14.0 aarch64 [upgradable from: 2.13.1-6]
freetype/stable 2.12.0 aarch64 [upgradable from: 2.11.0]
fribidi/stable 1.0.11-1 aarch64 [upgradable from: 1.0.10]
frida-python/root 15.1.12 aarch64 [upgradable from: 14.2.14-1]
frida-server/root 15.1.12 aarch64 [upgradable from: 14.2.14-1]
frida-tools/root 15.1.12 aarch64 [upgradable from: 14.2.14-1]
garcon/x11 4.16.1-2 aarch64 [upgradable from: 4.16.1-1]
gawk/stable 5.1.1-1 aarch64 [upgradable from: 5.1.0-1]
gdb/stable 10.1-6 aarch64 [upgradable from: 10.1-1]
gdbm/stable 1.22 aarch64 [upgradable from: 1.20]
gdk-pixbuf/stable 2.42.8-1 aarch64 [upgradable from: 2.42.6]
gettext/stable 0.21-5 aarch64 [upgradable from: 0.21-4]
git/stable 2.35.2 aarch64 [upgradable from: 2.34.0]
glib-bin/stable 2.72.0-1 aarch64 [upgradable from: 2.66.1-1]
glib/stable 2.72.0-1 aarch64 [upgradable from: 2.66.1-1]
googletest/stable 1.11.0-1 aarch64 [upgradable from: 1.11.0]
gpgv/stable 2.3.4 aarch64 [upgradable from: 2.3.1-1]
grep/stable 3.7-2 aarch64 [upgradable from: 3.6]
gtk-update-icon-cache/x11 3.24.33-1 aarch64 [upgradable from: 3.24.24-4]
gtk2/x11 2.24.32-35 aarch64 [upgradable from: 2.24.32-33]
gtk3/x11 3.24.33-1 aarch64 [upgradable from: 3.24.24-4]
gzip/stable 1.12 aarch64 [upgradable from: 1.10-4]
harfbuzz/stable 4.2.0 aarch64 [upgradable from: 2.8.2]
hicolor-icon-theme/x11 0.17-28 all [upgradable from: 0.17-27]
imlib2/stable 1.7.2-2 aarch64 [upgradable from: 1.7.2-1]
inetutils/stable 1.9.4-13 aarch64 [upgradable from: 1.9.4-11]
iso-codes/x11 4.6.0-2 all [upgradable from: 4.6.0-1]
jsoncpp/stable 1.9.5 aarch64 [upgradable from: 1.9.4]
krb5/stable 1.19.3 aarch64 [upgradable from: 1.19.2-1]
ldns/stable 1.8.1-1 aarch64 [upgradable from: 1.7.1-3]
less/stable 590 aarch64 [upgradable from: 563]
libandroid-shmem/stable 0.4 aarch64 [upgradable from: 0.3]
libandroid-support/stable 28-2 aarch64 [upgradable from: 28]
libaom/stable 3.3.0 aarch64 [upgradable from: 3.2.0]
libarchive/stable 3.6.1 aarch64 [upgradable from: 3.5.1]
libc++/stable 23b-3 aarch64 [upgradable from: 21d-1]
libcairo/stable 1.16.0-9 aarch64 [upgradable from: 1.16.0-7]
libcroco/stable 0.6.13-7 aarch64 [upgradable from: 0.6.13-5]
libcrypt/stable 0.2-5 aarch64 [upgradable from: 0.2-3]
libcurl/stable 7.82.0-1 aarch64 [upgradable from: 7.78.0]
libdav1d/stable 1.0.0 aarch64 [upgradable from: 0.9.2-1]
libdb/stable 18.1.40-3 aarch64 [upgradable from: 18.1.40-2]
libdrm/x11 2.4.107-1 aarch64 [upgradable from: 2.4.105-1]
libepoxy/x11 1.5.10 aarch64 [upgradable from: 1.5.7-1]
libexif/stable 0.6.24 aarch64 [upgradable from: 0.6.22]
libexpat/stable 2.4.8 aarch64 [upgradable from: 2.4.1-1]
libfontenc/x11 1.1.4-21 aarch64 [upgradable from: 1.1.4-20]
libgcrypt/stable 1.10.1 aarch64 [upgradable from: 1.9.3]
libglade/x11 2.6.4-22 aarch64 [upgradable from: 2.6.4-20]
libgmp/stable 6.2.1-1 aarch64 [upgradable from: 6.2.1]
libgnutls/stable 3.7.3 aarch64 [upgradable from: 3.6.16]
libgpg-error/stable 1.45 aarch64 [upgradable from: 1.42]
libgraphite/stable 1.3.14-1 aarch64 [upgradable from: 1.3.14]
libjpeg-turbo/stable 2.1.3 aarch64 [upgradable from: 2.1.0]
libllvm/stable 14.0.0 aarch64 [upgradable from: 12.0.1]
liblua53/stable 5.3.5-10 aarch64 [upgradable from: 5.3.5-6]
liblzo/stable 2.10-3 aarch64 [upgradable from: 2.10-2]
libmnl/stable 1.0.5 aarch64 [upgradable from: 1.0.4-1]
libmpfr/stable 4.1.0-1 aarch64 [upgradable from: 4.1.0]
libnghttp2/stable 1.47.0 aarch64 [upgradable from: 1.44.0]
libnl/stable 3.5.0-1 aarch64 [upgradable from: 3.5.0]
libnotify/x11 0.7.9-13 aarch64 [upgradable from: 0.7.9-11]
libopus/stable 1.3.1-5 aarch64 [upgradable from: 1.3.1-4]
libpcap/stable 1.10.1-1 aarch64 [upgradable from: 1.10.1]
libpixman/stable 0.40.0-2 aarch64 [upgradable from: 0.40.0-1]
libprotobuf/stable 2:3.19.3 aarch64 [upgradable from: 2:3.17.3]
librsvg/stable 2.54.0 aarch64 [upgradable from: 2.50.7]
libsoxr/stable 0.1.3-3 aarch64 [upgradable from: 0.1.3-2]
libsqlite/stable 3.38.2 aarch64 [upgradable from: 3.36.0-1]
libssh2/stable 1.10.0-2 aarch64 [upgradable from: 1.9.0-2]
libtiff/stable 4.3.0-2 aarch64 [upgradable from: 4.3.0]
libtirpc/stable 1.3.2-1 aarch64 [upgradable from: 1.3.1]
libunistring/stable 1.0 aarch64 [upgradable from: 0.9.10-4]
libusb/stable 1.0.26-1 aarch64 [upgradable from: 1.0.24-1]
libuuid/stable 1.0.3-5 aarch64 [upgradable from: 1.0.3-4]
libuv/stable 1.44.1 aarch64 [upgradable from: 1.41.1]
libvidstab/stable 1.1.0-2 aarch64 [upgradable from: 1.1.0]
libvorbis/stable 1.3.7-1 aarch64 [upgradable from: 1.3.7]
libwebp/stable 1.2.2 aarch64 [upgradable from: 1.2.0]
libwnck/x11 3.36.0-13 aarch64 [upgradable from: 3.36.0-11]
libx11/stable 1.7.2-1 aarch64 [upgradable from: 1.7.2]
libx264/stable 1:0.161.3049-1 aarch64 [upgradable from: 1:0.161.3049]
libxcomposite/x11 0.4.5-20 aarch64 [upgradable from: 0.4.5-19]
libxcursor/stable 1.2.0-20 aarch64 [upgradable from: 1.2.0-18]
libxdamage/x11 1.1.5-19 aarch64 [upgradable from: 1.1.5-18]
libxfce4ui/x11 4.17.2 aarch64 [upgradable from: 4.16.0-2]
libxfce4util/x11 4.17.1-5 aarch64 [upgradable from: 4.16.0-2]
libxfixes/stable 5.0.3-25 aarch64 [upgradable from: 5.0.3-23]
libxfont2/x11 2.0.5-1 aarch64 [upgradable from: 2.0.4-17]
libxi/stable 1.7.10-23 aarch64 [upgradable from: 1.7.10-21]
libxinerama/x11 1.1.4-25 aarch64 [upgradable from: 1.1.4-24]
libxkbfile/x11 1.1.0-22 aarch64 [upgradable from: 1.1.0-21]
libxklavier/x11 5.4-21 aarch64 [upgradable from: 5.4-19]
libxml2/stable 2.9.13 aarch64 [upgradable from: 2.9.12]
libxmu/x11 1.1.3-23 aarch64 [upgradable from: 1.1.3-22]
libxrandr/stable 1.5.2-24 aarch64 [upgradable from: 1.5.2-22]
libxshmfence/x11 1.3-29 aarch64 [upgradable from: 1.3-28]
libxxf86vm/x11 1.1.4-30 aarch64 [upgradable from: 1.1.4-29]
littlecms/stable 2.13.1-1 aarch64 [upgradable from: 2.12]
lsof/stable 4.94.0-1 aarch64 [upgradable from: 4.94.0]
lxde-icon-theme/x11 0.5.1-22 all [upgradable from: 0.5.1-21]
m4/stable 1.4.19-3 aarch64 [upgradable from: 1.4.18-3]
make/stable 4.3-2 aarch64 [upgradable from: 4.3-1]
mesa/x11 21.3.7-4 aarch64 [upgradable from: 17.3.9-34]
nano/stable 6.2 aarch64 [upgradable from: 5.8]
ndk-sysroot/stable 23b-7 aarch64 [upgradable from: 21d-2]
nmap-ncat/stable 7.92-2 aarch64 [upgradable from: 7.91-3]
nmap/stable 7.92-2 aarch64 [upgradable from: 7.91-3]
obconf/x11 2.0.4-28 aarch64 [upgradable from: 2.0.4-26]
openbox/x11 3.6.1-54 aarch64 [upgradable from: 3.6.1-51]
openjpeg/stable 2.4.0-1 aarch64 [upgradable from: 2.4.0]
openssh/stable 8.8p1-3 aarch64 [upgradable from: 8.8p1]
openssl-tool/stable 3.0.2 aarch64 [upgradable from: 1.1.1l]
openssl/stable 3.0.2 aarch64 [upgradable from: 1.1.1l]
pango/stable 1.50.6 aarch64 [upgradable from: 1.48.7]
pcre2/stable 10.39-2 aarch64 [upgradable from: 10.37]
pcre/stable 8.45-1 aarch64 [upgradable from: 8.45]
perl/stable 5.34.1-1 aarch64 [upgradable from: 5.34.0]
pinentry/stable 1.2.0 aarch64 [upgradable from: 1.1.1]
pkg-config/stable 0.29.2-1 aarch64 [upgradable from: 0.29.2]
pypanel/x11 2.4-35 aarch64 [upgradable from: 2.4-33]
python2-six/x11 1.12.0-23 all [upgradable from: 1.12.0-22]
python2-xlib/x11 0.24-23 all [upgradable from: 0.24-21]
python2/stable 2.7.18-12 aarch64 [upgradable from: 2.7.18-7]
python/stable 3.10.4 aarch64 [upgradable from: 3.9.6-3]
qemu-user-x86-64/stable 1:6.1.0-11 aarch64 [upgradable from: 1:6.1.0-9]
rhash/stable 1.4.2-2 aarch64 [upgradable from: 1.4.2]
sed/stable 4.8-2 aarch64 [upgradable from: 4.8-1]
shared-mime-info/x11 2.1-7 aarch64 [upgradable from: 2.1-2]
sqlcipher/stable 4.5.0-2 aarch64 [upgradable from: 4.5.0]
sqlite/stable 3.38.2 aarch64 [upgradable from: 3.36.0-1]
startup-notification/x11 0.12-27 aarch64 [upgradable from: 0.12-26]
tcl/stable 8.6.11-1 aarch64 [upgradable from: 8.6.11]
termux-auth/stable 1.4-2 aarch64 [upgradable from: 1.4]
termux-keyring/stable 3.1 all [upgradable from: 1.9]
termux-licenses/stable 2.0-2 all [upgradable from: 2.0-1]
termux-tools/stable 0.174 all [upgradable from: 0.130]
thunar/x11 4.17.7-4 aarch64 [upgradable from: 4.16.6-1]
tigervnc/x11 1.10.1-23 aarch64 [upgradable from: 1.10.1-16]
tsu/stable 8.6.0 all [upgradable from: 8.5.1-3]
util-linux/stable 2.38 aarch64 [upgradable from: 2.37]
vim-runtime/stable 8.2.4650 all [upgradable from: 8.2.2800]
vim/stable 8.2.4650 aarch64 [upgradable from: 8.2.2800]
w3m/stable 0.5.3.20190105-8 aarch64 [upgradable from: 0.5.3.20190105-6]
wget/stable 1.21.3-2 aarch64 [upgradable from: 1.21.1]
xcb-util/x11 0.4.0-28 aarch64 [upgradable from: 0.4.0-27]
xfce4-panel/x11 4.16.3-18 aarch64 [upgradable from: 4.15.2-15]
xfce4-session/x11 4.16.0-7 aarch64 [upgradable from: 4.16.0-2]
xfce4-settings/x11 4.16.2-4 aarch64 [upgradable from: 4.16.1-1]
xfce4/x11 2.1-16 all [upgradable from: 2.0-14]
xfconf/x11 4.16.0-5 aarch64 [upgradable from: 4.16.0-2]
xfdesktop/x11 4.16.0-4 aarch64 [upgradable from: 4.16.0-2]
xfwm4/x11 4.16.1-5 aarch64 [upgradable from: 4.16.1-2]
xkeyboard-config/x11 2.32-2 all [upgradable from: 2.32-1]
xorg-font-util/x11 1.3.2-21 aarch64 [upgradable from: 1.3.2-20]
xorg-fonts-75dpi/x11 1.0.3-26 all [upgradable from: 1.0.3-25]
xorg-fonts-alias/x11 1.0.4-4 all [upgradable from: 1.0.4-3]
xorg-fonts-encodings/x11 1.0.5-23 all [upgradable from: 1.0.5-22]
xorg-iceauth/x11 1.0.8-25 aarch64 [upgradable from: 1.0.8-24]
xorg-mkfontscale/x11 1.2.1-23 aarch64 [upgradable from: 1.2.1-22]
xorg-xauth/x11 1.1-22 aarch64 [upgradable from: 1.1-21]
xorg-xkbcomp/x11 1.4.5-2 aarch64 [upgradable from: 1.4.5-1]
xorg-xrdb/x11 1.2.0-21 aarch64 [upgradable from: 1.2.0-20]
xorg-xsetroot/x11 1.1.2-26 aarch64 [upgradable from: 1.1.2-25]
xxhash/stable 0.8.1 aarch64 [upgradable from: 0.8.0-1]
zlib/stable 1.2.12 aarch64 [upgradable from: 1.2.11-5]
zstd/stable 1.5.2 aarch64 [upgradable from: 1.5.0]
Android version:
11
Kernel build information:
Linux localhost 4.19.113-perf-g4a878ec38421 #1 SMP PREEMPT Tue May 11 02:13:41 CST 2021 aarch64 Android
Device manufacturer:
Xiaomi
Device model:
Redmi K30 5G

```

**Comment by xtkoba** [comment]
Your `python` is too old. Please update it. Also I recommend `pkg up` to update everything.

**Comment by xtkoba** [comment]
If you still encounter a similar problem with the latest python, then please open a new issue. Thanks.

**Comment by Young-Lord** [comment]
The problem solved after upgrading python to 3.10.4. Thanks for your replying!
